rest-service/manager_rest/deployment_update/manager.py

```diff
@@ -591,7 +591,8 @@ def _execute_workflow(self,
             is_system_workflow=False
         )
         if deployment:
```

Comment:
Pretty sure this if-clause is not necessary, `deployment` is being used as if it isn't `None` in previous lines.